packages/cli/src/commands/upgrade/__tests__/upgrade.test.ts

```diff
@@ -215,18 +208,22 @@ test('fetches regular patch, adds remote, applies patch, installs deps, removes
     success Upgraded React Native to v0.58.4 🎉. Now you can review and commit the changes"
     `);
   expect(
-    snapshotDiff(samplePatch, (fs.writeFileSync: any).mock.calls[0][1], {
-      contextLines: 1,
-    }),
+    snapshotDiff(
+      samplePatch,
+      (fs.writeFileSync as jest.Mock).mock.calls[0][1],
+      {
+        contextLines: 1,
+      },
+    ),
   ).toMatchSnapshot(
     'RnDiffApp is replaced with app name (TestApp and com.testapp)',
   );
 }, 60000);
 test('fetches regular patch, adds remote, applies patch, installs deps, removes remote when updated from nested directory', async () => {
   mockFetch(samplePatch, 200);
-  (execa: any).mockImplementation(mockExecaNested);
+  ((execa as unknown) as jest.Mock).mockImplementation(mockExecaNested);
```

Comment:
I usually make it `((execa as any) as jest.Mock)`, which might be a little bit smaller. It doesn't really matter here :P